local/kaltura/yui/build/moodle-local_kaltura-ltipanel/moodle-local_kaltura-ltipanel-debug.js

```diff
@@ -297,8 +308,8 @@ Y.extend(LTIPANELMEDIAASSIGNMENT, Y.Base, {
                 srcNode : Y.Node.create('<div id="dialog" />'),
                 headerContent : '',
                 bodyContent : this.panelbodycontent,
-                width : this.panelwidth+"px",
-                height : this.panelheight+"px",
+                width : panelwidth,
+                height : panelheight+"px",
```

Comment:
why was "px" removed for width but not for height? makes the code very hard to track...